Yoast/yoast-components
# Add a fullWidth property to the Configuration Wizard steps to configu…

…re the content width.

## Summary

This PR can be summarized in the following changelog entry:

* Adds a "fullWidth" prop to the Configuration Wizard steps and use it to configure the steps content width

## Relevant technical choices:

* Needs https://github.com/Yoast/wordpress-seo/pull/7823 for the plugin

## Test instructions

This PR can be tested by following these steps:

* on the standalone version, just `yarn start` and go to http://localhost:3333/
* check the first step has no max-width set on the container `yoast-wizard-content-container`, while all the following steps should have a max-width
* for the real-scenario case, with real content, see test instructions on https://github.com/Yoast/wordpress-seo/pull/7823 for the plugin

Fixes #259
